Source/WebCore/css/parser/CSSSelectorParser.cpp

```diff
@@ -723,13 +723,9 @@ std::unique_ptr<CSSParserSelector> CSSSelectorParser::consumePseudo(CSSParserTok
 
     std::unique_ptr<CSSParserSelector> selector;
 
-    if (colons == 1) {
-        selector = CSSParserSelector::parsePseudoClassSelector(token.value());
-        if (!selector)
-            return nullptr;
```

Comment:
Note to reviewer: we early return when selector is null just after the else ends, so this is OK.